### PR TITLE
Templatedsites + kiraSites

### PR DESCRIPTION
**Pull request by ianyimi**

### CODE REVIEW

File: src/pages/kirax23/alto.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import dynamic from "next/dynamic";
+import { ScrollData } from "../../themes/Alto/types/scroll";
```
Comment (alex-shortt) [inline src/pages/kirax23/alto.tsx:4]: dont need `../../`, just `themes/`

File: src/pages/kirax23/index.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import { NextPage } from "next";
 import Head from "next/head";
 import dynamic from "next/dynamic";
-import { GothamProps } from "../themes/Gotham";
+import { GothamProps } from "../../themes/Gotham";
```
Comment (alex-shortt) [inline src/pages/kirax23/index.tsx:4]: dont need ../../, just themes/

File: src/themes/Alto/models/AltoModel.tsx [hunk 3]
```diff
@@ -58,19 +58,21 @@ export default function Model(props: JSX.IntrinsicElements["group"]) {
...
+  //   () =>
+  //     new MeshStandardMaterial({
+  //       map: treeTex,
+  //       transparent: true,
+  //       alphaMap: treeTranspTex,
+  //       side: THREE.DoubleSide,
+  //       depthWrite: false,
+  //     }),
+  //   [treeTranspTex, treeTex]
+  // );
```
Comment (alex-shortt) [inline src/themes/Alto/models/AltoModel.tsx:75]: comments

File: src/scenes/Alto/index.tsx [hunk 4]
```diff
@@ -8,6 +8,7 @@ import Alto, { AltoProps } from "themes/Alto";
 import Lighting from "themes/Alto/components/Lighting";
 import Dropoff from "themes/Alto/components/Dropoff";
 import { HDRI } from "themes/components/HDRBackground";
+import Video from "spacesvr";
```
Comment (alex-shortt) [inline src/scenes/Alto/index.tsx:11]: bad import

File: src/themes/Alto/components/Scrolls/Scroll.tsx [hunk 5]
```diff
@@ -1,11 +1,19 @@
...
+  Suspense,
+  useLayoutEffect,
+  useMemo,
+  useEffect,
+} from "react";
 import * as THREE from "three";
 import { Text } from "@react-three/drei";
 // @ts-ignore
 import { animated, useSpring } from "react-spring/three";
+import { useLimiter } from "../../../../scenes/Silks/utils/limiter";
```
Comment (alex-shortt) [inline src/themes/Alto/components/Scrolls/Scroll.tsx:16]: import from spacesvr

File: src/themes/Alto/kira.tsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+};
+
+const Alto = (props: Partial<AltoProps>) => {
+  const [scrollCount, setScrollCount] = useState(0);
+
+  return (
+    <AltoContext.Provider value={{ ...defaultContext, ...props }}>
+      <Suspense fallback={null}>
+        <AltoModel />
+        <Grass />
```
Comment (alex-shortt) [inline src/themes/Alto/kira.tsx:61]: why do you need this file? looks the same as the standard alto

File: src/themes/Alto/components/Scrolls/Scroll.tsx [hunk 7]
```diff
@@ -44,28 +55,54 @@ const Scroll = (props: JSX.IntrinsicElements["group"] & ScrollProps) => {
...
+      inner.current.lookAt(camera.position);
+      if (!open) setOpen(true);
+    } else {
+      if (open) setOpen(false);
     }
   });
 
+  useEffect(() => {
+    if (visits < 2) {
+      if (inRange) {
```
Comment (alex-shortt) [inline src/themes/Alto/components/Scrolls/Scroll.tsx:91]: &&?

File: src/scenes/Gotham/OldGotham.tsx [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+        speed: 1.7,
+      }}
+    >
+      <Sky inclination={sunPos} distance={night ? 0 : 1000000} />
+      {stars && <Stars count={1500} fade />}
+      {fogColor && <Fog color={new THREE.Color(fogColor)} near={10} far={80} />}
+      {audio && (
+        <Audio url={audio} position={new Vector3(-6, 1, 2.5)} volume={1.2} />
+      )}
+      <Lighting color={lightColor} />
```
Comment (alex-shortt) [inline src/scenes/Gotham/OldGotham.tsx:65]: seems like the only thing this audio adds is dual environment and lighting. if thats the case it should be a scene. in fact we already have a kira scene, think you should be able to edit that

**Review by alex-shortt — CHANGES_REQUESTED** [review]
- please fill out the pr description a little
- remove assets folder, assets should be on s3
- seems like you should be able to use the kira scene to add lighting if that's the main thing

File: src/scenes/Gotham/OldGotham.tsx (hunk 8 above)
Comment (ianyimi) [reply src/scenes/Gotham/OldGotham.tsx:65]: it also removes the buidings and adds the outside mesh with the heightmap and gradient applied

File: src/themes/Alto/kira.tsx (hunk 6 above)
Comment (ianyimi) [reply src/themes/Alto/kira.tsx]: it doesnt have the sun or the fog for the hdri thats the only diff

File: src/pages/kirax23/alto.tsx [hunk 9]
```diff
@@ -0,0 +1,61 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import dynamic from "next/dynamic";
+import { ScrollData } from "themes/Alto/types/scroll";
+
+const Standard = dynamic(import("scenes/Alto/kira"), { ssr: false });
```
Comment (alex-shortt) [inline src/pages/kirax23/alto.tsx:6]: scenes should have a capital, Kira

File: src/themes/Alto/kira.tsx [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+import { Interactable, Video } from "spacesvr";
+import Grass from "./components/Grass";
+import Scrolls from "./components/Scrolls";
+import Tablatures from "./components/Tablatures";
+import Birds from "./components/Birds";
+import AudioReactive from "./components/AudioReactive";
+import { ScrollData } from "./types/scroll";
+import { Preload } from "@react-three/drei";
+import { AltoContext } from "./index";
+
```
Comment (alex-shortt) [inline src/themes/Alto/kira.tsx:13]: capital K for Kira

**Review by alex-shortt — CHANGES_REQUESTED** [review]
small thing then good

**Review by alex-shortt — APPROVED** [review]

**Pull request merged**